doc/source/ray-air/examples/serving_guide.ipynb

```diff
@@ -0,0 +1,923 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(air-serving-guide)=\n",
+    "\n",
+    "# Serve Ray AIR Predictor with `ModelWrapper`\n",
+    "\n",
+    "[Ray Serve](rayserve) is the recommended tool to deploy models trained with AIR.\n",
+    "\n",
+    "The core concept is the `ModelWrapper`. The `ModelWrapper` takes a [predictor](ray.air.predictor.Predictor) class and a [checkpoint](ray.air.checkpoint) and transforms them into a live HTTP endpoint. \n",
+    "\n",
+    "We'll start with a simple quick-start demo showing how the `ModelWrapper` fits into Ray AIR."
```

Comment:
showing how you can use the ModelWrapper to deploy your model for online inference.